src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Android.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_PlatformDoesNotSupportNiFiles Condition="'$(Platform)' == 'arm'">true</_PlatformDoesNotSupportNiFiles>
+    <_PlatformDoesNotSupportNiFiles Condition="'$(Platform)' == 'armel'">true</_PlatformDoesNotSupportNiFiles>
+    <_PlatformDoesNotSupportNiFiles Condition="'$(Platform)' == 'arm64'">true</_PlatformDoesNotSupportNiFiles>
+  </PropertyGroup>
+  <ItemGroup>
+    <NativeBinary Include="$(BinDir)libcoreclr.so" />
+    <NativeBinary Condition="'$(_PlatformDoesNotSupportNiFiles)' != 'true'" Include="$(BinDir)libcoreclrtraceptprovider.so" />
```

Comment:
This check is incorrect. __PlatformDoesNotSupportNiFiles_ should be used for platforms where we cannot generate NI images right now. They just happen to be same as where this binary does not build.

If you are able to build this for Android, please fix this check accordingly so that it can be included for that platform.

FYI - @hqueue @hseok-oh @jyoungyun